blocks/productcards/productcards.css

```diff
@@ -57,7 +56,8 @@ h1#accessories-title strong {
 }
 
 @media screen and (min-width: 1024px) {
-  .productcards-container {
+  .card-heading.productcards-container {
```

Comment:
Why this change?

Reply:
Because it matches the instance in line 12.

Reply:
i meant the whole change - why remove the width declaration above, and add it on this breakpoint?

Reply:
See screenshot at https://github.com/hlxsites/airlessco/issues/158#issuecomment-1442452870